hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/BaseFileOnlyRelation.scala

```diff
@@ -81,7 +81,9 @@ case class BaseFileOnlyRelation(override val sqlContext: SQLContext,
     super.imbueConfigs(sqlContext)
     // TODO Issue with setting this to true in spark 332
     if (HoodieSparkUtils.gteqSpark3_4 || !HoodieSparkUtils.gteqSpark3_3_2) {
-      sqlContext.sparkSession.sessionState.conf.setConfString("spark.sql.parquet.enableVectorizedReader", "true")
```

Comment:
should not use contains. 
spark always inject this config.

The reason why this place is forcibly set to true is because the parent class of BaseFileOnlyRelationship previously forced set spark.sql.parquet.enableVectorizedReader=false.  users unable to turn on vectorization, resulting in a significant decrease in query performance. So at that time, it was forcibly set to true in this location.

however https://issues.apache.org/jira/browse/HUDI-3639 This PR removes the behavior of the parent class of BaseFileOnlyRelationship.
Therefore, we no longer need to force set spark.sql.parquet.enableVectorizedReader=true in BaseFileOnlyRelationship; Whether to enable vectorization is left to the user for decision

Reply:
just remove  line 83. to line 85

Reply:
Thanks for reminder. Updated.